canvas_modules/harness/cypress/support/canvas/verification-cmds.js

```diff
@@ -226,7 +226,10 @@ Cypress.Commands.add("verifyNodeElementWidth", (nodeName, nodeElement, width) =>
 			const nodeElementSelector = "[data-id='" + node[0].getAttribute("data-id").replace("grp", nodeElement) + "']";
 			cy.get(nodeElementSelector)
 				.invoke("css", "width")
-				.should("eq", width);
+				.then((cssValue) => {
+					// width should be in the compare range of cssValue
+					expect(Number(width.split("px")[0])).to.be.closeTo(Number(cssValue.split("px")[0]), Cypress.env("compareRange"));
```

Comment:
I think this should be a utility function or a cypress function.  Something that takes 2 pixel sizes and compares them.